### PR TITLE
Add handle voting power changed tests for Kyber Governance

### PR DESCRIPTION
**Pull request by manhlx3006**

### CODE REVIEW

File: test/kyberGovernance.js [hunk 1]
```diff
@@ -1385,30 +1385,273 @@ contract('KyberGovernance', function (accounts) {
...
+      await checkVoteDataChange(proposalId, totalVotes, options, voteCounts, voter2, 0, 0);
+
+      await votingStrategy.callbackWithdrawal(governance.address, voter1, newVotingPower, [proposalId]);
+      totalVotes = newVotingPower;
+      voteCounts = updateVoteCountsOnVotingPowerChanges(voteCounts, 1, voter1VotingPower, newVotingPower);
+      await checkVoteDataChange(proposalId, totalVotes, options, voteCounts, voter1, newVotingPower, 1);
+      await checkVoteDataChange(proposalId, totalVotes, options, voteCounts, voter2, 0, 0);
+    });
+
+    it('no changes when voter has not voted yet', async () => {
```
Comment (ducquangkstn) [inline test/kyberGovernance.js:1572]: duplicate with line from 1535?

File: test/kyberGovernance.js [hunk 2]
```diff
@@ -1385,30 +1385,273 @@ contract('KyberGovernance', function (accounts) {
...
+      await checkVoteDataChange(proposalId, totalVotes, options, voteCounts, voter1, voter1VotingPower, 1);
+      await checkVoteDataChange(proposalId, totalVotes, options, voteCounts, voter2, 0, 0);
+
+      let newVotingPower = new BN(10).pow(new BN(10));
+      await votingStrategy.callbackWithdrawal(governance.address, voter2, newVotingPower, [proposalId]);
+      // no changes
+      await checkVoteDataChange(proposalId, totalVotes, options, voteCounts, voter1, voter1VotingPower, 1);
+      await checkVoteDataChange(proposalId, totalVotes, options, voteCounts, voter2, 0, 0);
+
+      await votingStrategy.callbackWithdrawal(governance.address, voter1, newVotingPower, [proposalId]);
```
Comment (ducquangkstn) [inline test/kyberGovernance.js:1565]: pls add event check here

File: test/kyberGovernance.js [hunk 3]
```diff
@@ -1385,30 +1385,273 @@ contract('KyberGovernance', function (accounts) {
...
+      );
+      await governance.submitVote(proposalId, 1, {from: voter1});
+      let totalVotes = voter1VotingPower;
+      voteCounts = updateVoteCountsOnOptionChanges(voteCounts, 0, 1, voter1VotingPower);
+
+      await checkVoteDataChange(proposalId, totalVotes, options, voteCounts, voter1, voter1VotingPower, 1);
+      await checkVoteDataChange(proposalId, totalVotes, options, voteCounts, voter2, 0, 0);
+
+      let newVotingPower = new BN(10).pow(new BN(10));
+      await votingStrategy.callbackWithdrawal(governance.address, voter2, newVotingPower, [proposalId]);
```
Comment (ducquangkstn) [inline test/kyberGovernance.js:1560]: Pls add a check that event is not emitted

**Review by ducquangkstn — APPROVED** [review]
LGTM. just need minor changes

**Review by Anyhowclick — APPROVED** [review]

**Pull request merged**